src/Redux/reducers/Greeting/greeting.js

```diff
@@ -0,0 +1,26 @@
+const RANDOM_GREET_ACTION = 'Greeting/greeting/RANDOM_GREET_ACTION';
+
+export const generateGreeting = () => (dispatch) => {
+  const myHeaders = new Headers();
+  myHeaders.append('Content-Type', 'application/json');
+  myHeaders.append('Accept', 'application/json');
+  fetch('http://localhost:4000/greeting', {
```

Comment:
- [ ] Kindly notice that you should write the API URL right `http://localhost:400/greeting`.

Reply:
Can you elaborate more on this. @ShahierNashaat. What do you mean by `right`, currently my rails is running on local host with port 4000. How should I rephrase it?

Reply:
As you wrote on your Readme file to choose the 400 port but you wrote 4000 in the front end, it doesn't matter but please update the readme file at the back end repo.

Reply:
I see—ill correct that. Thank you @ShahierNashaat

Reply:
Fixed with [3eaa1f7](https://github.com/Tadesse-Alemayehu/Hello-Rails-Back-End/pull/1/commits/3eaa1f78e5a792649883798b5075be2638901b12) commit hash.